README.md

```diff
@@ -8,18 +8,51 @@
 
 > An ActionCable companion for Y.js clients
 
-This is a monorepo with [npm modules](./packages) and [Ruby gems](./gems).
-Please check the respective sub-repos for more information. 
+TODO: Explain what this library does.
```

Comment:
@eliias I added some TODOs. This is where I need your help but I wanted to provide extension points already. We should tackle these in follow-up PRs. The most important thing to understand when coming to a repository is why it exists in the first place.